cms/static/js/factories/videos_index.js

```diff
@@ -10,10 +10,26 @@ define(
             uploadButton,
             previousUploads
         ) {
+            function refreshList() {
+                $.ajax({
+                    url: postUrl,
+                    contentType: 'application/json',
+                    dataType: 'json',
+                    type: 'GET'
+                }).done(function(responseData) {
```

Comment:
@tanmaykm What happens if we get error on ajax call ? I would like to see a jasmine test for that scenario.